Examples/Tests/electrostatic_sphere_eb/analysis_rz_mr.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python
+
+# Copyright 2024 Olga Shapoval, Edoardo Zoni
+#
+# This file is part of WarpX.
+#
+# License: BSD-3-Clause-LBNL
+
+# This script tests the embedded boundary in RZ.
+# A cylindrical surface (r=0.1) has a fixed potential 1 V.
+# The outer surface has 0 V fixed.
+# Thus the analytical solution has the form:
+# phi(r) = A+B*log(r), Er(r) = -B/r.
+
```

Comment:
Could you add a quick summary explaining how this differs from `analysis_rz.py` (i.e. what aspects of MR does this check?)

Reply:
Also, could you confirm that this test fails with the current `development` version of WarpX?

Reply:
When I started working on this PR I separated the analysis script into two because it had been modified to process the openpmd results of the new test (which uses openpmd only), thus failing on the plotfile results of the old test (which uses plotfile only). I can try to see if it is possible to re-merge them together.

Reply:
And yes, I just ran the test with the latest `development` branch and the analysis fails as the error on the radial electric field on level 1 is larger than the tolerance.
```
At level = 0 :
max error of phi =  0.004078274696046572
max error of Er =  0.0025379720096999625
tolerance =  0.0041
At level = 1 :
max error of phi =  0.00028625211446039984
max error of Er =  1.0028845814970835
tolerance =  0.0041
Traceback (most recent call last):
  File "/tmp/ci-gymyMO8XZo/rt-WarpX/WarpX-tests/2024-07-17/ElectrostaticSphereEB_RZ_MR/./analysis_rz_mr.py", line 109, in <module>
    get_error_per_lev(ts,level)
  File "/tmp/ci-gymyMO8XZo/rt-WarpX/WarpX-tests/2024-07-17/ElectrostaticSphereEB_RZ_MR/./analysis_rz_mr.py", line 104, in get_error_per_lev
    assert(errmax_Er < tolerance)
AssertionError
```